Cargo.lock

```diff
@@ -357,7 +357,7 @@ dependencies = [
  "deno_typescript 0.34.0",
  "dirs 2.0.2 (registry+https://github.com/rust-lang/crates.io-index)",
  "dlopen 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
- "dprint-plugin-typescript 0.6.0 (registry+https://github.com/rust-lang/crates.io-index)",
+ "dprint-plugin-typescript 0.6.1 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
run cargo build to update the lock file

Reply:
Ok, thanks for reply. I am installing cargo, since this is my friend's device.